metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -363,9 +404,10 @@ void convert_batch_to_row_diff(const std::string &pred_succ_fprefix,
 
     // total number of set bits in the original rows
     std::vector<uint32_t> row_nbits_block;
+    std::vector<uint32_t> row_nbits_block_other;
```

Comment:
add a comment to describe what this is supposed to be storing any how it's related to `row_nbits_block`